lark-stubs/load_grammar.pyi

```diff
@@ -24,3 +24,6 @@ class GrammarBuilder:
     def validate(self) -> None: ...
 
     def build(self) -> Grammar: ...
+
+
+def find_grammar_errors(text: str, start: str='start'): ...
```

Comment:
Adding a return type is a good idea I think

Reply:
You're not wrong. Added now.